packages/components/src/List/ListToVirtualizedList/ListToVirtualizedList.component.js

```diff
@@ -111,6 +114,7 @@ ListToVirtualizedList.propTypes = {
 };
 ListToVirtualizedList.defaultProps = {
 	displayMode: 'table',
+	cellDictionary: {},
```

Comment:
This is not necessary, you only spread it, and spreading undefined is supported.